### DISCUSSION TITLE
How to change RadioButton size(width/height) ?

### DISCUSSION THREAD

**Discussion by sparrow2015** [AvaloniaUI/Avalonia#14691] [Q&A] [answered]
```
<RadioButton ></RadioButton>
<RadioButton Width="16" Height="16"></RadioButton>
```
Result:
![image](https://github.com/AvaloniaUI/Avalonia/assets/16752460/9f77f1d9-d44b-479b-a8de-becb44c83a0c)

After I set Width and Height, the icon of RadioButton is not completely circle.
Is there someone help me?

**Comment by maxkatz6** [comment, marked as answer]
If you want to make it smaller, you probably need to restyle the control. You can base your control theme/styles on top of default one - https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/RadioButton.xaml.

Side note: we probably should put glyph in the Viewbox, so resizing would be easier.

**Reply by sparrow2015** [reply]
Thank you, I will try soon.
